### DISCUSSION TITLE
Cron job not invoked -- how to debug?

### DISCUSSION THREAD

**Discussion by asindhu** [vercel/next.js#64704] [Help]
### Summary

I configured a cron job to run daily, my vercel.json file looks like this:

`{
  "crons": [ { "path": "/api/cron", "schedule": "30 2 * * *" } ]
}`

This job is simple, it runs some calculations and saves an entry into a third-party external persistent database, so I can see the result of each successful run as a new row added to my database each day around the same time. Was all working just fine for about a month. But, I've just recently realized that as of 4/10, something broke, because nothing's been added to the database since then.

Absolutely nothing has changed. I haven't pushed any new code to Vercel or changed anything about this project or the cron job script, nor have I changed anything about the external database.

I assumed maybe there's some issue that cropped up with the script itself causing it to fail even if the cron job is invoked correctly --  but I ran it manually and it still works just fine, and adds a new row to the database as expected each time I run it manually.

Next I changed the timing of the cron job in the vercel.json to match a time in the near future (making sure to account for the UTC time difference) and re-deployed the project. At the expected time, nothing shows up in my database. I have a few logs in my script, which I don't see anywhere on the Vercel side. I assume I should see some kind of logs when the cron job is invoked? There's just no signs of life whatsoever and I'm not sure how else I can debug what's going on or why the cron job is not being invoked. Any ideas?

**Comment by chreds** [comment]
Did you figure out a way to debug at all? I am running into the same thing.

**Reply by asindhu** [reply]
Never found a way to debug it, unfortunately... but eventually something I did got it working again. All I did was mess with the cron file a few times, change a few random things in my code base, and deploy again, and then a few days later it started working. A bit frustrating to have so little visibility into if/when it works or doesn't...

**Reply by migaval** [reply]
which vercel plan do you have? It's not working for me either
